kubernetes/slackbridge.yml.erb

```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: deployment
+  labels:
+    app: slackbridge
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: slackbridge
+  template:
+    metadata:
+      labels:
+        app: slackbridge
+    spec:
+      containers:
+        - name: slackbridge
+          image: "docker.ocf.berkeley.edu/slackbridge:<%= version %>"
+          resources:
+            limits:
+              memory: 2048Mi
```

Comment:
I don't think it should need this much memory, [currently it has 512 MB](https://github.com/ocf/services/blob/ca9253c8a82ba6e021cfc9d326b6c084fcf965cb/apps/slackbridge/app.yaml#L5) and that's been fine (as far as I can tell).